Classes/Format/Alto.php

```diff
@@ -42,4 +46,49 @@ public function getRawText(\SimpleXMLElement $xml)
         }
         return $rawText;
     }
+
+
+
+    /**
+     * This extracts the fulltext data from ALTO XML and returns it in MiniOCR format
+     *
+     * @access public
+     *
+     * @param \SimpleXMLElement $xml: The XML to extract the raw text from
+     *
+     * @return string The unformatted fulltext in MiniOCR format
+     */
+    public function getTextAsMiniOcr(\SimpleXMLElement $xml)
+    {
+        $rawText = '';
+        $xml->registerXPathNamespace('alto', 'http://www.loc.gov/standards/alto/ns-v2#');
```

Comment:
There are already documents with version 3 and 4. If I'm not mistaken Zeitungsportal has mostly them. How are we going to handle those documents?

Reply:
Of course, there is ALTO v3 and v4. But this is a another issue (#488) which has to be fixed. Maybe by different Alto classes with different "getTextAsMiniOcr()" methods.

For the DDB-Zeitungsportal is no problem here as we do not index into their Solr.

Reply:
Good point! Thanks! Then there is no problem.